content/en/user-manual/animation/anim-state-graph-assets.md

```diff
@@ -68,8 +68,8 @@ The available transition variables are:
 
 | Variable            | Description |
 |---------------------|-------------|
-| Duration            | The duration of the animation in seconds. |
-| Exit Time           | If provided, this transition will only be active for the exact frame during which the source states progress passes the time specified. Given as a normalized value of the source state's duration. Values less than 1 will be checked every animation loop. |
+| Duration            | The duration of the transition in seconds. |
+| Exit Time           | The time at which to exit the source state and enter the destination state. Given in normalized time based on the source state\'s duration. Providing no value allows the source state to exit with this transition at any time. A value of less than 1 will make the transition available for exit at that time during every loop of the source state. |
```

Comment:
I don't _think_ you need to escape the apostrophe? Try it without and let me know if I'm wrong.

Reply:
Ok, I've removed it.